src/store/modules/ui.store.js

```diff
@@ -253,5 +267,11 @@ export default {
         setImportFile(state, flagValue) {
             state.importFile = flagValue
         },
+        setHeaderHeight(state, height) {
+            state.headerHeight = height
+        },
+        setMenuTrayWidth(state, width) {
+            state.menuTrayWidth = width
+        },
```

Comment:
you should also add the corresponding action above (and do all the type checking, `parseFloat` and what not in the action)